packages/fela/index.d.ts

```diff
@@ -1,13 +1,44 @@
-export interface Renderer {
-    clear(): void;
-    renderRule(rule: Function, props?: any): string;
-    renderKeyframe(keyframe: any, props?: any): string;
-    renderFont(family: string, files: any[], properties?: any): string;
-    renderStatic(style: string | any, selector?: string): string;
+declare module "fela" {
+
+  import React from 'react';
+
+  type TRuleProps = {};
+  type TRule = (props: TRuleProps) => IStyle; 
+  type TKeyFrame = TRule;
+  type TRendererCreator = (config?: IConfig) => IRenderer;
+  type TPlugin = (style: IStyle) => IStyle; //http://fela.js.org/docs/advanced/Plugins.html
+  type TEnhancer = (renderer: IRenderer) => IRenderer; //http://fela.js.org/docs/advanced/Enhancers.html
+
+  interface IRenderer {
+    renderRule(rule: TRule, props: TRuleProps): void;
+    renderKeyframe(keyFrame: TKeyFrame, props: TRuleProps): void;
+    renderFont(family: string, files: Array<string>, props: TRuleProps): void;
+    renderStatic(style: string, selector?: string): void;
     renderToString(): string;
-    subscribe(callback: Function): any;
+    subscribe(event: (msg: string) => void): { unsubscribe: () => void; }
```

Comment:
Looks great!
Just the subscribe seems wrong. It gets called with change objects, not a string. 
e.g. `{type: 1, selector: '.a', declaration: 'color:red' }`